pkg/martaapi/taxonomy.go

```diff
@@ -209,3 +204,23 @@ var LineDirections = map[Line][]Direction{
 	Gold:  []Direction{North, South},
 	Red:   []Direction{North, South},
 }
+
+//Termini allow for lookups of all the terminuseses of the different lines
+var Termini = map[Line]map[Direction]Station{
+	Green: map[Direction]Station{
+		East: EdgewoodCandlerParkStation,
```

Comment:
reel dum nitpick:  For the `Station` type, we could remove the word `Station` each `const` to reduce stuttering